lookup_tables/ipinfo/ipinfo_asn_detections_engine.yml

```diff
@@ -0,0 +1,292 @@
+AnalysisType: lookup_table
+LookupName: ipinfo_asn_detections_engine
```

Comment:
I got feedback from Security and I agree, can we remove suffix `detections_engine` from the 2 LUTs (and the helpers). Then the key in the JSON is `ipinfo_asn` which looks nicer.